requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy
-tensorflow
-pillow
-flask_cors
+numpy==1.16.2
+tensorflow==1.13.1
+pillow==5.4.1
+flask_cors==3.0.7
```

Comment:
Is this required since it should be part of `max-base`?

Reply:
Nope, thanks